components/list/list-item-button-mixin.js

```diff
@@ -29,17 +29,20 @@ export const ListItemButtonMixin = superclass => class extends ListItemMixin(sup
 			button {
 				background-color: transparent;
 				border: none;
+				color: unset;
 				cursor: pointer;
 				display: block;
+				font-family: unset;
+				font-size: unset;
+				font-weight: unset;
+				letter-spacing: unset;
+				line-height: unset;
 				height: 100%;
 				outline: none;
+				padding: 0;
+				text-align: start;
```

Comment:
This is a bit gross - overriding `button` defaults that are impacted the typography etc styles on the content now that the content is within the button